.github/workflows/python.yml

```diff
@@ -1,4 +1,4 @@
-on: [push, pull_request,repository_dispatch]
```

Comment:
why removing pull_request?

Reply:
Since this workflow is a step in pull-request.yml so not needed here.
Kept separate pull-request.yml to add more steps in future (e.g. C unit test)

Reply:
got it.